plugin/parser/differ/mysql/differ.go

```diff
@@ -277,6 +312,86 @@ func (*SchemaDiffer) SchemaDiff(oldStmt, newStmt string) (string, error) {
 		}
 	}
 
+	// We compare the CREATE TRIGGER/EVENT/FUNCTION/PROCEDURE statements based on strcmp.
+	var newNodeStmt []string
+	var inplaceDropStmt []string
+	var inplaceAddStmt []string
+
+	functionMap, err := buildUnsupportObjectMap(oldUnsupportFilterStmts, function)
```

Comment:
We might want to return all four maps from buildUnsupportObjectMap() altogether. If leftover statements exist, we should return an error as unrecognized statements.